skbio/diversity/tests/test_driver.py

```diff
@@ -635,5 +636,148 @@ def test_get_beta_diversity_metrics_sorted(self):
         n = sorted(list(m))
         self.assertEqual(m, n)
 
+
+class TestPartialBetaDiversity(TestCase):
+    def setUp(self):
+        self.table1 = [[1, 5],
+                       [2, 3],
+                       [0, 1]]
+        self.sids1 = list('ABC')
+        self.tree1 = TreeNode.read(io.StringIO(
+            '((O1:0.25, O2:0.50):0.25, O3:0.75)root;'))
+        self.oids1 = ['O1', 'O2']
+
+        self.table2 = [[23, 64, 14, 0, 0, 3, 1],
+                       [0, 3, 35, 42, 0, 12, 1],
+                       [0, 5, 5, 0, 40, 40, 0],
+                       [44, 35, 9, 0, 1, 0, 0],
+                       [0, 2, 8, 0, 35, 45, 1],
+                       [0, 0, 25, 35, 0, 19, 0]]
+        self.table2 = np.array(self.table2)
```

Comment:
Can you remove this line so that `table2` is a nested Python list? That would have caught a bug where `counts` is `array_like` and gets converted in `_validate_counts_matrix`.